examples/domain_write/domain_write_mpi.cpp

```diff
@@ -100,7 +100,7 @@ int main(int argc, char **argv)
      * http://computationalradiationphysics.github.io/libSplash/classsplash_1_1_domain_collector.html
      * for more information on this interface.
      **/
-    dc.writeDomain(10,                    /* timestamp/iteration */
+    dc.writeDomain(10,                    /* iteration step (timestep in case of PIConGPU) */
```

Comment:
iteration ~~step (timestep in case of PIConGPU)~~

libSplash users do not know PIConGPU